tsconfig.json

```diff
@@ -1,17 +1,16 @@
 {
-  "include": ["./src/**/*.ts"],
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "declaration": true,
-    "declarationMap": true,
     "esModuleInterop": true,
-    "lib": ["esnext"],
-    "module": "esnext",
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["ES2020"],
+    "module": "CommonJS",
```

Comment:
Getting an error from Rollup. It looks like `module` and `moduleResolution` need to remain as `esnext`, not CommonJS, at least in this PR.

Reply:
I tried to fix Rollup, but it seems like it's more effort than it's worth at this point since I'm planning to remove it in the next PR. Changing `module` and `moduleResolution` alone doesn't seem to fix it at least.